components/common/posts/PostsFeed.tsx

```diff
@@ -216,8 +216,7 @@ function ImageAltText({ alt }: { alt: string }) {
   )
 }
 
-// @TODO record embeds
-function PostEmbeds({ item }: { item: AppBskyFeedDefs.FeedViewPost }) {
```

Comment:
This diff looks super complex but the overview is that we just moved out the rendering for embed record into it's own component so that it can be used inside message context viewer.